### DISCUSSION TITLE
Linux: no access to automounted SMB volume

### DISCUSSION THREAD

**Discussion by steamfan** [enzo1982/freac#653] [Help & Support]
Hello,
I'm using Ubuntu 24 and my music directory is located on a NAS. The music directory is mounted as a SMB drive via automounter. Unfortunately freac does no see the automounted drive; other applications (file manger Dolphin, Thunar or terminal) have full access to that drive.
Am I doing something wrong?
What further information do you need to investigate into that issue?
Regards
Michael

**Comment by ghostrider77** [comment]
Hi Michael,

I had a similar problem. I have an auto-mounted HDD for storing data in general, whilst my OS and my home folder is on an SDD. freac was not able to write to the HDD, it did not even see it complaining about permission errors. Not sure how I solved this years ago, but after an OS reinstall I faced again with this problem.

If you installed freac with snap (app center), then it is the likely cause of the problem, since snap essentially creates a sandbox for the applications with limited amount of permissions. If this is the problem indeed, then you either uninstall the snap-installed freac and use the appimage from the freac website, or you may try to make the snap-version work with the following command: 
```
snap connect freac:removable-media
```
